src/pages/hotel/booking/booking-confirmation/hotel-overview/HotelOverview.scss

```diff
@@ -1,38 +1,60 @@
 .main-content-hotel {
-  margin-top: 15px;
-  width: 705px;
-  margin-left: 3px;
-  padding: 2px;
 }
 .book-image-parent {
   min-width: 250px;
 }
 .hotel-content {
   border: 1px solid rgba(220, 220, 220, 1);
-  padding: 1px;
-  height: 14rem;
```

Comment:
current hotel image styles changed for large screens because of this. instead of changing the current styles completly, better to keep the current styles for large screens as such for a min width and you can add additional styles for small screens.

Reply:
ok